yaml/generator.go

```diff
@@ -90,8 +90,7 @@ func (c *TestCase) getTemplateVars() (string, map[string]any) {
 	case "java":
 		return c.javaTemplateVars()
 	default:
-		ginkgo.Fail("unknown generator " + generator)
-		return "", nil
+		return filepath.FromSlash("./docker-compose-" + generator + "-template.yml"), map[string]any{}
```

Comment:
Could test cases be added for this? Where is this compose file located / what happens when it doesn't exist?

Reply:
I believe the file is expected to be in the current path of where the test is run (e.g. it's NOT in the TESTCASE_BASE_PATH.)  For Beyla (which imports oats as a Go module,) I believe this is going to require having its own copy; I don't think I can test that before merging it.  (I did try to run a java generator in Beyla to see if it would try current working directory, but it required some java things before looking for the docker compose template file.)
If the file doesn't exist, it errors out with a file not found error that is explained with sufficient context.